File: Problem-019/solution.js
function countingSundays(firstYear, lastYear) {
  let count = 0;
  for (let year = firstYear; year <= lastYear;year ++){
    for ( let month = 0; month< 12; month ++){
      const date = new Date( year, month, 1);
      if (date.getDay() === 0){
        count ++;

      }
    }
  }
  return count;
}

File: Problem-019/solution.py
import datetime

def counting_sundays():
    count = 0

    for year in range(1901, 2001):
        for month in range(1, 13):

            date = datetime.date(year, month, 1)

            if date.weekday() == 6:
                count += 1

    return count


print(counting_sundays())
